src/Mod/Arch/ArchWindow.py

```diff
@@ -1333,16 +1334,33 @@ def getSubVolume(self,obj,plac=None):
                         width = max(b.XLength,b.YLength,b.ZLength)
         if not width:
             width = 1.1112 # some weird value to have little chance to overlap with an existing face
+
         if not base:
-            return None
+            if Draft.isClone(obj,"Window"):  # if this object has not base, check whether this is a clone and use the original's base
+                if hasattr(obj,"CloneOf"):
+                    orig = obj.CloneOf
+                else:
+                    orig = obj.Objects[0]  # not sure what is this exactly
```

Comment:
This is when the object that is cloning a window is not another Arch object, but a Draft clone.